h/static/scripts/test/annotation-metadata-test.js

```diff
@@ -80,8 +67,7 @@ describe('annotation-metadata', function () {
       context('when there is no document.title', function() {
         it('returns the domain as the title', function() {
           var model = {
-            document: {},
-            uri: 'http://example.com/',
+            uri: 'http://example.com/'
```

Comment:
Small nit here and in a few other places, the last entry in a multi-line array or object literal should have a  trailing comma:

```js
var foo = [
  entry1,
  entry2,
]
```

This reduces noise in diffs by not changing the content of a line depending on whether it is the last entry or not.